### PR TITLE
Automated review system

### PR DESCRIPTION
**Pull request by Akshiteshnaithani**
Targets #3167 

### Description
This task aims to automate the review system by implementing scheduled tasks using cron jobs. The current manual process of generating assessment cards for employees will be replaced with an automated system that generates assessment cards at regular intervals(Quarterly).

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: app/Console/Kernel.php [hunk 1]
```diff
@@ -28,7 +29,8 @@ class Kernel extends ConsoleKernel
         EndedProject::class,
         FixedBudgetProject::class,
         NotificationToProjectTeamMembersToUpdateEffortOnGoogleChat::class,
-        JobExpiredEmailToHr::class
+        JobExpiredEmailToHr::class,
+        SystemReviewQuaterly::class
```
Comment (Ayush8923) [inline app/Console/Kernel.php:33]: Please update all places accordingly
```suggestion
        QuarterlyReviewSystemForEmployee::class
```

File: Modules/HR/Console/SystemReviewQuaterly.php [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+use Modules\HR\Entities\IndividualAssessment;
+
+class SystemReviewQuaterly extends Command
+{
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'hr:quarterly-review-system';
```
Comment (Ayush8923) [inline Modules/HR/Console/SystemReviewQuaterly.php:18]:
```suggestion
    protected $name = 'employee:quarterly-review-system-for-employee';
```

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
Added some comments. Please check. Also I think, we don't need the `EmployeeController`, so please revert this file.

**Comment by Akshiteshnaithani** [comment]
> Added some comments. Please check. Also I think, we don't need the `EmployeeController`, so please revert this file.

CI checks were getting fail due to `EmployeeController`  file.

**Comment by Ayush8923** [comment]
As per the offline discussion, please use SRP(Single responsibility principal) principal.

**Comment by Akshiteshnaithani** [comment]
> As per the offline discussion, please use SRP(Single responsibility principal) principal.

Implemented the SRP principal.

**Comment by Akshiteshnaithani** [comment]
### New Requirement :
currently we have added a `Review Status` filter in our Working staff page so that we could filter out all the employees whose review is pending or in-progress or completed. Now we need to remove that filter and whenever I will land onto the working staff page, all the employee should show up whose review is pending or in-progress or completed

**Review by Bahugunajii — APPROVED** [review]

**Pull request merged**